app/app.go

```diff
@@ -1352,10 +1364,9 @@ func (app *Evmos) setupUpgradeHandlers() {
 	case v12.UpgradeName:
 		// no store upgrades
 	case v13.UpgradeName:
-	// no store upgrades
-
-	// !! ATTENTION !!
+		// no store upgrades
```

Comment:
This comment should be removed? Is it referring to the context below or above?

Reply:
I just moved the `!! ATTENTION!!` into two lines below